### ISSUE TITLE
Login for Pro users

### ISSUE THREAD

**Issue by Gabe-LS** [bunnykek/WeTransferTool#3]
Hello! I'd like to try WeTransferTool (thanks for developing it!) but I need the possibility to log into my account before uploading the file(s). Is there any chance to add that feature?

**Comment by bunnykek** [comment]
Yeah, its possible. I have that already planned but currently I'm busy.

**Comment by Gabe-LS** [comment]
Ok! I look forward to trying the new feature when it is available — no rush, of course.
